flink-table/flink-sql-client/src/test/resources/sql/select.q

```diff
@@ -50,8 +50,38 @@ java.lang.IllegalArgumentException: testing elements of values source shouldn't
 # (we can't test changelog mode and table mode in IT case)
 # ==========================================================================
 
+SET table.dml-sync=true;
+[INFO] Session property has been set.
+!info
+
+create table MyTable (
+  id int,
+  str string
+) with (
+  'connector' = 'filesystem',
+  'path' = '$VAR_PATH',
+  'format' = 'csv'
+);
+[INFO] Table has been created.
+!info
+
+SELECT id, COUNT(*) as cnt, COUNT(DISTINCT str) as uv
+FROM MyTable
+GROUP BY id;
++----+-------------+----------------------+----------------------+
+| op |          id |                  cnt |                   uv |
++----+-------------+----------------------+----------------------+
+Received a total of 0 row
+!ok
+
+INSERT INTO MyTable SELECT * FROM (VALUES (1, 'Hello World'), (2, 'Hi'), (2, 'Hi'), (3, 'Hello'), (3, 'World'), (4, 'ADD'), (5, 'LINE'));
+[INFO] Submitting SQL update statement to the cluster...
+[INFO] Table update statement has been successfully submitted to the cluster:
+
```

Comment:
Would be better to print some information about waiting result. What will Hive print when executing INSERT INTO?

Reply:
![image](https://user-images.githubusercontent.com/33114724/112153296-15591800-8c1e-11eb-8f9e-def19f8b4457.png)

Reply:
In beeline, hive doesn't have any warning or info...

Reply:
Yes. Otherwise, users don't know whether the job is completed or submitted async.

Reply:
![image](https://user-images.githubusercontent.com/33114724/112716812-89980200-8f23-11eb-8660-f0a39f88047a.png)